dia/diameter_3gpp_ts29_061_sgi.dia

```diff
@@ -24,6 +24,7 @@
 @inherits diameter_rfc4005_nasreq
 @inherits diameter_rfc4006_cc
 @inherits diameter_3gpp_base
+@inherits diameter_travelping
```

Comment:
can we choose a better name where the prefix `_travelping` will not be appear?

Reply:
why should we? This is a vendor specific dictionary, including the vendors name is only natural.

Reply:
ok